modules/monitoring/images.yml

```diff
@@ -52,15 +52,15 @@ images:
       - registry.sighup.io/fury/kiwigrid/k8s-sidecar
 
   - name: Grafana [Fury Monitoring Module]
-    source: docker.io/grafana/grafana
-    tag:
-      - "6.6.2"
-      - "7.1.5"
-      - "7.3.6"
-      - "7.5.7"
-      - "8.0.6"
-      - "8.1.2"
-      - "8.3.3"
+    source: grafana
+    build:
+      context: custom/grafana
+      args:
+        - name: GRAFANA_VERSION
```

Comment:
I would drop this arg in favor of the tag: array, see the comment on the `Dockerfile`

Reply:
Done. ✅